src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Revisions.cshtml

```diff
@@ -46,8 +46,13 @@
             @foreach (var revision in Model.Review.Revisions)
             {
                 <li class="list-group-item">
-                    @revision.Name <span class="font-italic" date="@revision.CreationDate"></span>
-                    <form asp-resource="@Model.Review" asp-requirement="@ReviewOwnerRequirement.Instance" class="form-inline float-right" asp-page-handler="Delete" method="post">
+                    @revision.Name 
+                    <span class="font-italic pl-3">
+                        <span date="@revision.CreationDate"></span>
+                        <span author="@revision.Author"></span>
```

Comment:
you can avoid tag helper here all together:
`<span>by @revision.Author</span>`

I used tag helper for date because it has a lot of logic.

Reply:
I wanted to not show "by" if the author was not populated.

Reply:
@if (revision.Author != null) {
<span>by @revision.Author</span>

}

Why not auto-popultate the author for the old revisions?